kong/plugins/prometheus/api.lua

```diff
@@ -1,11 +1,12 @@
+local Schema = require("kong.db.schema")
 local prometheus = require "kong.plugins.prometheus.exporter"
 
-local consumers_schema = kong.db.consumers.schema
+local empty_schema = Schema.new({ fields = {} })
 
 return {
 
   ["/metrics"] = {
-    schema = consumers_schema, -- not used, could be any schema
+    schema = empty_schema,
```

Comment:
A comment describing why specifying a schema is necessary would still be helpful to future readers of this code

Reply:
Added the comment a bit before (it does not appear on this particular diff)